Casks/k/kindle-create.rb

```diff
@@ -0,0 +1,30 @@
+cask "kindle-create" do
+  version "1.93.0.0"
+  sha256 :no_check
+
+  url "https://d2bzeorukaqrvt.cloudfront.net/KindleCreateInstaller.pkg",
+      verified: "d2bzeorukaqrvt.cloudfront.net/"
+  name "Kindle Create"
+  desc "Creating beautiful books has never been easier"
+  homepage "https://www.amazon.com/Kindle-Create/b?node=18292298011"
+
+  livecheck do
+    skip "No version information available"
+  end
+
+  auto_updates true
+
+  depends_on macos: ">= :catalina"
+
+  pkg "KindleCreateInstaller.pkg"
+
+  uninstall pkgutil:  "com.amazon.kc",
+
+  zap trash: [
+    "~/Library/Saved Application State/com.amazon.kc.savedState",
+  ]
```

Comment:
```suggestion
  auto_updates true
  depends_on macos: ">= :catalina"

  pkg "KindleCreateInstaller.pkg"

  uninstall pkgutil: "com.amazon.kc"

  zap trash: "~/Library/Saved Application State/com.amazon.kc.savedState"
```

We still have to fix the style-related CI failures. Run `brew style --fix kindle-create.rb` to have them corrected automatically.